src/BPEssentials/Commands/Chat/Sudo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BrokeProtocol.Entities;
+using BPEssentials.Utils;
+using BPEssentials.Abstractions;
+
+namespace BPEssentials.Commands
+{
+    public class Sudo : Command
+    {
+        public void Invoke(ShPlayer player, ShPlayer target, string message)
+        {
+            Core.Instance.Logger.LogInfo($"[SUDO] {target.username}: {message}");
+            ChatUtils.SendToAllEnabledChat(ChatUtils.FormatMessage(target, message), false);
```

Comment:
Why not invoke the `SvPlayer.SvGlobalChatMessage` method on the target? This will allow sudo to be executed on commands as well and won't require an extra log message. [Like in an older version of BPE.](https://github.com/BPEssentials/Core/blob/c2ed63a760e6136996c0964843d8a6d4a856d08e/source/BP-Essentials/Chat/Commands/Admin/Sudo.cs#L28)

Reply:
OK, I will change that, seems better